packages/components/src/styleConstants.ts

```diff
@@ -29,4 +29,5 @@ export const fonts = {
   SERIF: `"Spectral", serif`,
   SANS_SERIF: `"Libre Franklin", sans-serif`,
   MONOSPACE: `font-family: “SF Mono”, “Segoe UI Mono”, “Roboto Mono”, “Ubuntu Mono”, Menlo, Courier, monospace;`,
+  COURIER: `Courier, sans-serif`,
```

Comment:
You should check with Nguyet, but I think the Monospace font is what she wants to use for addresses and numbers.

At least that's the font-stack she gave for those.